eidas_node/tests/proxy_service/test_views.py

```diff
@@ -205,6 +205,20 @@ def test_create_light_response_cannot_find_request(self):
         with cast(TextIO, (DATA_DIR / 'saml_response.xml').open('r')) as f:
             view.saml_response = SAMLResponse(parse_xml(f.read()), 'relay123')
 
+        with self.assertRaisesMessage(SecurityError, 'Invalid in-response-to id'):
+            view.create_light_response('test-light-response-issuer')
+
+    def test_create_light_response_cannot_find_request(self):
+        view = IdentityProviderResponseView()
+        view.request = self.factory.post(self.url)
+        view.storage = IgniteStorage('test.example.net', 1234, 'test-proxy-service-request-cache', '')
+        self.cache_mock.get.return_value = None
+
+        with cast(TextIO, (DATA_DIR / 'saml_response.xml').open('r')) as f:
+            # SAML ID is prefixed with T for cases when the original id starts wit a digit
+            data = f.read().replace('test-saml-request-id', 'Ttest-saml-request-id')
```

Comment:
It is probably better to modify the file itself.

Reply:
Done.